example/satosa/integration_test/main_ec.py

```diff
@@ -0,0 +1,283 @@
+import requests
```

Comment:
please do not duplicate similar code, replace the main with this one where EC is used by default or create an argparse prompt where options can be used to defined if using RSA or EC or which alg to use